.github/workflows/go.yml

```diff
@@ -60,7 +60,6 @@ jobs:
         run: make build
 
       - name: Check that fresh version of RPC bindings is committed (for minimum supported Go version only)
-        if: ${{ matrix.go_versions == '1.19' }}
```

Comment:
It's not version-dependent, I agree, but the idea was to check it only once, we don't need to perform it three times in a raw.

But if you'd like to, then comment should also be adjusted: `(for minimum supported Go version only)`.

Reply:
And BTW, I think that in some very special cases it may be version-dependent due to https://github.com/nspcc-dev/neo-go/issues/3133.

Reply:
The cost of this check is negligible compared to `make build` itself, so. I'll fix the comment.

Reply:
And still, you do this three times instead of one. And have three identical (failing or successful) test reports instead of one report. So this doesn't look good to me.

Reply:
Although it's not a separate job, this check is bound to the building job, so OK.